spec/lib/mcb/commands/courses/show_spec.rb

```diff
@@ -22,6 +22,7 @@
   let(:course) { create(:course, name: 'P', provider: provider, subjects: [subject1], site_statuses: [site_status1]) }
   let(:rolled_over_course) do
     new_course = course.dup
+    new_course.update(start_date: course.start_date - 1.year)
```

Comment:
Ugh, I don't think we should be having to do this to fix this test. We should be able to create a course that is in the next recruitment cycle instead of having to manually roll-over a course. Probably a new card at this point.

Reply:
I've just fixed this and pushed up a commit. About the same effort as making a ticket.

Reply:
Cool. Looks good to me. I'll reflect these changes in the applications open PR too 👍